docs/getting-started-kubernetes.rst

```diff
@@ -156,142 +156,256 @@ Below are deployment examples, which the ``values.yaml`` file provided to the He
 Network Operator Deployment with RDMA Shared Device Plugin
 ----------------------------------------------------------
 
-Network operator deployment with the default version of the OFED driver and a single RDMA resource mapped to ens1f0 netdev.:
+First install the Network Operator with NFD enabled:
 
-``values.yaml`` configuration file for such a deployment:
+``values.yaml``:
 
 .. code-block:: yaml
 
     nfd:
       enabled: true
-    sriovNetworkOperator:
```

Comment:
Is the default `false` for sriovNetworkOperator?

Reply:
Yes - https://github.com/mellanox/network-operator/blob/56e4a72472f7746cece5513468e6386605b4d729/deployment/network-operator/values.yaml#L25-L26